test/mocha/mock.plugin.js

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = {
+  type: 'service',
+  api: {
+    index: async ({createIndexes, collections}) => {
```

Comment:
Should this be called `createIndexes` instead of `index` or perhaps there should be another level like `create` under `index`? It doesn't seem to follow the naming convention for functions unless this is meant to mean "this function will index things". But that doesn't seem to *quite* match up with what it's doing.

Reply:
`createIndexes` is a possibility, but I'm concerned about confusion related to the helper function being passed in by the same name.  How about `expandIndexes` or 'createCustomIndexes`?

Reply:
I think `expandIndexes` would be good.